scripts/remove-whitespace.js

```diff
@@ -0,0 +1,14 @@
+var fs = require('fs');
+
+const someFile = process.argv[2];
+console.log(someFile);
+fs.readFile(someFile, 'utf8', function (err, data) {

```

Comment:
duplisering av det som ligger i `remove-old-features.js`

Reply:
Trenger et skript som kan rydde opp i gamle tester som kan inneholde UTF-8 koder. Når jeg kopierer inn features fra NVA-scenarioer så fjerner jeg UTF-8 koder, men de kan ligge igjen i javascript-filene for tester som er skrevet tidligere.

Reply:
men samme kode trenger ikke å være 2 plasser?
mener at dette scriptet kan kalle samme metode som `getDirectories()` gjør i `remove-old-features.js`